app/components/Reactions/Reaction.tsx

```diff
@@ -53,9 +59,17 @@ const Reaction = ({
     return <></>;
   }
 
+  let tooltipContent = emoji + '\n';
```

Comment:
Could we maybe do "x, y og z reagerte med ...", sort of similar to how GitHub does it. Not sure how I feel about the emoji name just being next to all the names.